odmantic/model.py

```diff
@@ -499,6 +501,29 @@ def __new__(
 
 
 BaseT = TypeVar("BaseT", bound="_BaseODMModel")
+TraversalStateT = NamedTuple(
```

Comment:
Not sure this should have a `T` suffix - it's not a `TypeVar` like above, just a straight named tuple.

Reply:
Good point, I'll change it